### ISSUE TITLE
Default the "reduce UI animations" preference to true if prefers-reduced-motion is true

### ISSUE THREAD

**Issue by nolanlawson** [nolanlawson/pinafore#695]
The default should be

```js
window.matchMedia('(prefers-reduced-motion: reduce)').matches
```

For users who have configured their OS/browser this way.
